test/run

```diff
@@ -429,43 +398,34 @@ function run_dockerfiles_test() {
   rm "$dockerfile_tmp"
 }
 
+TEST_SUMMARY=''
+ct_enable_cleanup
+
 CID_FILE_DIR=$(mktemp -d)
 
 build_image "test-app"
 
 # Run the chosen tests
-TEST_LIST=${TEST_LIST_APP} run_all_tests "test-app"
+TEST_SET=${TEST_LIST_APP} ct_run_tests_from_testset "test-app"
 
 build_image "start-hook-test-app"
 
 # Run the chosen tests
-TEST_LIST=${TEST_LIST_HOOK_APP} run_all_tests "start-hook-test-app"
+TEST_SET=${TEST_LIST_HOOK_APP} ct_run_tests_from_testset "start-hook-test-app"
 
 build_image "perl-test-app"
 
 # Run the chosen tests
-TEST_LIST=${TEST_LIST_PERL_APP} run_all_tests "perl-test-app"
+TEST_SET=${TEST_LIST_PERL_APP} ct_run_tests_from_testset "perl-test-app"
 
 # Run the rest of the tests by defining TEST_LIST to include the rest of the relevant cases
-TEST_LIST=run_dockerfiles_test
+TEST_SET=run_dockerfiles_test
 
 # These tests only make sense for rhscl
 if [ "$OS" == "rhel7" ] || [ "$OS" == "centos7" ]; then
-  # autocleanup only enabled here as only the following tests so far use it
-  ct_enable_cleanup
-  TEST_LIST="$TEST_LIST test_scl_enable"
+  TEST_SET="$TEST_SET test_scl_enable"
 fi
 
 # Execute the rest of the tests depending on what TEST_LIST includes now
-run_all_tests
-
-# All tests executed, see the results
-echo "$test_short_summary"
-
-if [ $TESTSUITE_RESULT -eq 0 ] ; then
-  echo "Tests for ${IMAGE_NAME} succeeded."
-else
-  echo "Tests for ${IMAGE_NAME} failed."
-fi
-
-exit $TESTSUITE_RESULT
+ct_run_tests_from_testset "rest-of-the-tests"
```

Comment:
What is the name 'rest-of-the-tests' used for?

I guess, 'run_dockerfiles_test' and 'test_scl_enable' can be used as TEST_REST_LIST variable or name it like 'dockerfiles-scl-enable-tests'

Reply:
addressed in new commit.